src/compressor/CMakeLists.txt

```diff
@@ -23,8 +23,13 @@ if(HAVE_BROTLI)
 endif()
 
 add_library(compressor STATIC ${compressor_srcs})
-if(HAVE_QATZIP)
-  target_link_libraries(compressor PRIVATE ${QATZIP_LIBRARIES})
+if(HAVE_QATZIP AND HAVE_QATDRV)
+  include_directories("${QatDrv_INCLUDE_DIRS}")
```

Comment:
if QatDrv libraries already have "QatDrv_INCLUDE_DIRS" as their interface include directories, probably we can dispense with `include_directories("${QatDrv_INCLUDE_DIRS}")`? also, i don't think it's a good practice to add the include directory globally. better off adding it on per-target basis. and `target_link_libraries()` does this for us already.

Reply:
You are right in most scenarios. However, the interface include directories **qatzip.h**  of target **qatzip::qatzip** is including **cpa_dc.h** which is in QatDrv_INCLUDE_DIRS, see https://github.com/intel/QATzip/blob/master/include/qatzip.h. Here is the way I have tried but failed: add "QatDrv_INCLUDE_DIRS" as  interface include directories of **qatzip::qatzip**.
In addition, the QATZIP team said the line "#include <cpa_dc.h>" will be removed soon. At that time, this line "include_directories("${QatDrv_INCLUDE_DIRS}")" will also be deleted.

Reply:
You are right in most scenarios. However, the interface include directories **qatzip.h**  of target **qatzip::qatzip** is including **cpa_dc.h** which is in QatDrv_INCLUDE_DIRS, see https://github.com/intel/QATzip/blob/master/include/qatzip.h. Here is the way I have tried but failed: add "QatDrv_INCLUDE_DIRS" as  interface include directories of **qatzip::qatzip**.
In addition, the QATZIP team said the line "#include <cpa_dc.h>" will be removed soon. At that time, this line "include_directories("${QatDrv_INCLUDE_DIRS}")" will also be deleted.